client/logic/game/units/unit.gd

```diff
@@ -52,6 +52,8 @@ signal died
 var _possible_terrains_to_move_buffer: Array[Terrain]
 var _possible_terrains_to_move_calculating: bool
 
+var _types: GlobalTypes = Types
```

Comment:
can we make that when this is loaded, we preload already a variable selecting the ID? then we don't need to select the ID at all.

Reply:
For now, I would recommend keeping it as it is, since we are likely to replace it with a property abstraction in the future.